Misc/NEWS.d/next/Library/2024-08-05-10-52-51.gh-issue-122681.q0T58t.rst

```diff
@@ -0,0 +1,2 @@
+Merge ``m_atan2()`` and ``c_atan2()`` helper function, move to
+``Modules/_math.h``.
```

Comment:
I don't think that this change benefits to users, you can remove the NEWS entry.

Reply:
Done.  Could you, please, then add "skip news" label?